kubernetes/opencost.yaml

```diff
@@ -102,6 +104,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: opencost
+  namespace: opencost
```

Comment:
ClusterRoleBinding's are not namespaced?

Reply:
ClusterRoleBindings are not namespaced. If you put this in, delete the namespace, it's left behind. If you change the namespace and kubectl apply you get
```
clusterrolebinding.rbac.authorization.k8s.io/opencost unchanged
```